owasp-top10-2017-apps/a10/games-irados/app/routes.py

```diff
@@ -97,9 +99,11 @@ def newuser():
             message, success = database.insert_user(username, hashed_psw)
             if success == 1:
                 flash("Novo usuario adicionado!", "primary")
+                logging.info("\nEvent: %s - Timestamp: %s - Route: %s - Ip: %s - Username: %s\n" % ("New user registered!", datetime.datetime.now(), "/register", request.remote_addr, username))
                 return redirect('/login')
             else:
                 flash(message, "danger")
+                logging.error("\nEvent: %s - Timestamp: %s - Route: %s - Ip: %s - Username: %s\n" % ("Error registering new user!", datetime.datetime.now(), "/register", request.remote_addr, username))
                 return redirect('/register')
 
         flash("Passwords must be the same!", "danger")
```

Comment:
Hey, @gabriel-cantergiani! You are doing great adding better logs to all the main endpoints!

I believe you should log if `Passwords` are different at this point, there could be some shady business going on. What do you think?

Reply:
Yes, of course! Missed that one!